static/chats.js

```diff
@@ -191,3 +192,23 @@ ievadesLauks.addEventListener("keyup", function(event) {
     suutiZinju();
   }
 });
+
+let mutebtn = document.getElementById("mutebtn")
+mutebtn.addEventListener("click", bezSkanas)
+
+function bezSkanas(){
+  let statuss = mutebtn.value
+  if(statuss == "on") {
+    mutebtn.value = "off"
+    mutebtn.style.background = "url(static/img/speaker_mute.svg) no-repeat"    
+  } else {
+    mutebtn.value = "on"
+    mutebtn.style.background = "url(static/img/speaker.svg) no-repeat"
+  }
+}
+
+function skanja() {
+  let audio = new Audio('static/sounds/water_droplet.mp3')
+  if (mutebtn.value == "on") audio.play()
```

Comment:
It kā nav kritiski, bet varētu šo te audio objektu saukt tikai tad, ja vajag:
```js
if (mutebtn.value == "on") new Audio('static/sounds/water_droplet.mp3').play()
```

Reply:
Ok. Jo īsāk, jo labāk.
Man pagāja stunda, kamēr atradu, kā IF uzrakstīt vienā rindiņā :)

Reply:
Nebūt nē, īsāk labāk nav. Mana doma bija tāda, ka mēs varētu šo te new saukt tikai tad, ja vajag:
```js
if(mutebtn.value == "on") {
  const audio = new Audio('static/sounds/water_droplet.mp3')
  audio.play()
}